src/components/Display.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Display({ result }){
+    return(
+        <div className="result">
+            <p>{result}</p>
+        </div>
+    );
+}
+Display.propTypes = {
+    result: PropTypes.string.isRequired,
+};
+
```

Comment:
default value of `result` should be `'0'`, please validate this with `defaultProps` [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/DefaultProp.png)]